src/metric.js

```diff
@@ -25,9 +42,17 @@ var getRandomness = function (callback) {
         }
     }
   }
-  return new Promise(function (resolve) {
-    console.log('asking to reload to ', 8 * 8 * bloombits + num_definitions);
-    crypt.refreshBuffer(8 * 8 * bloombits + num_definitions, resolve);
+  var num_desired_rand_bytes = 8 * 8 * bloombits * num_definitions;
```

Comment:
Yup. The rest is logging I added to debug this.

Cheers,
$ ls
On Feb 11, 2016 6:11 PM, "Will" <notifications@github.com> wrote:

> In src/metric.js
> <https://github.com/willscott/freedomjs-anonymized-metrics/pull/15#discussion_r52683396>
> :
>
> > @@ -25,9 +42,17 @@ var getRandomness = function (callback) {
> >          }
> >      }
> >    }
> > -  return new Promise(function (resolve) {
> > -    console.log('asking to reload to ', 8 * 8 * bloombits + num_definitions);
> > -    crypt.refreshBuffer(8 * 8 * bloombits + num_definitions, resolve);
> > +  var num_desired_rand_bytes = 8 * 8 * bloombits * num_definitions;
>
> this seems to be the main change - multiplying by num_definitions right?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/willscott/freedomjs-anonymized-metrics/pull/15/files#r52683396>
> .
>